components/automate-ui/src/app/helpers/auth/regex.ts

```diff
@@ -24,7 +24,6 @@ export class Regex {
 
     // Allows valid FQDN only
     VALID_FQDN: /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-zA-Z0-9_]+([\-\.]{1}[a-zA-Z0-9]+)*\.[a-zA-Z]{2,5}(:[0-9]{1,5})?(\/.*)?$/,
-    VALID_DATA_FEED_URL: /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-zA-Z0-9_]+([\-\.]{1}[a-zA-Z0-9]+)*\.[a-zA-Z]{2,5}(:[0-9]{1,5})?(\/.*)?$/,
```

Comment:
We don't want to kill this because the whole application may use this regex

Reply:
thought you may have checked. already
@msorens - are we removing validation here forever?

Reply:
i dont think we have any need to validate URLs like this